src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -172,7 +186,7 @@ public abstract class EC2Cloud extends Cloud {
 
     private transient volatile AmazonEC2 connection;
 
-    protected EC2Cloud(String name, boolean useInstanceProfileForCredentials, String credentialsId, String privateKey, String sshKeysCredentialsId,
+    protected EC2Cloud(String name, boolean useInstanceProfileForCredentials, String credentialsId, String sshPrivateKey, String sshKeysCredentialsId,
```

Comment:
The fact that you could rename this parameter without making any other changes indicates that it was not actually used, so maybe delete the parameter assuming there are no external subclasses? Or at least rename it to something that indicates it is unused.

Reply:
Or just revert the change, which does not seem to have any purpose.